packages/board-server/tsconfig.json

```diff
@@ -20,7 +20,8 @@
     "useUnknownInCatchVariables": true,
     "experimentalDecorators": true,
     "useDefineForClassFields": false,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "esModuleInterop": true
```

Comment:
Curious, why did you need that?

Reply:
Again, my experience of TS/JS is fairly limited so perhaps there is a better solution, however the `node:sqlite` library which `better-sqlite3` uses uses commonjs - I couldn't find a sqlite library that didn't require some kind of commonjs support. Open to suggestions here.